sql/driver/driver_test.go

```diff
@@ -227,3 +227,60 @@ func TestInsecure(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestTransactions(t *testing.T) {
```

Comment:
I was originally planning to test that the transaction object gets passes through all these commands, but didn't get to it.